visualizer/actionButton.js

```diff
@@ -222,6 +232,7 @@
         // If the user is just changing an argument value, so we just
         // refresh the tree with new arguments values
         // TODO: update argument values
+        ohmEditor.semantics.emit('change:operationArgumentVals', args);
```

Comment:
Nit: I'd just call this "operationArguments" -- it's kind of redundant to say "argument values".